standards/data-destruction.md

```diff
@@ -0,0 +1,35 @@
+---
+expires: 2019-03-15
+---
+# Data Destruction
+
+'Data destruction' is the process of erasing or otherwise destroying data stored on virtual/electronic or physical mediums such as, but not limited to, tapes and hard disks in order to completely render data irretrievable and inaccessible and otherwise void.
+
+## The base principle
+
+For legislative, regulative, privacy and security purposes, it **must** be possible to decommission and delete (irreversibly ‘erase’ or ‘destroy’) data and confirm to a degree of relative confidence it has been completed.
+
+## Data lifecycle
+
+The following standards and guidelines are the basis for data decommissioning or destruction. Follow and apply them as appropriate. There might also be extra steps specific to a data set or system.
+
+- National Cyber Security Centre (NCSC) guidance on end-user device reset procedures: https://www.ncsc.gov.uk/guidance/end-user-device-guidance-factory-reset-and-reprovisioning
+- NCSC guidance on secure sanitisation of storage media: https://www.ncsc.gov.uk/guidance/secure-sanitisation-storage-media
+- NCSC Cloud Security Principle 2: Asset Protection and Resilience (Data Destruction): https://www.ncsc.gov.uk/guidance/cloud-security-principle-2-asset-protection-and-resilience#sanitisation
+- Payment Card Industry Data Security Standard (PCI-DSS) (Data Destruction): https://www.pcisecuritystandards.org
+
+Data Destruction **must** include, unless otherwise superseded by NCSC, PCI-DSS or specific MOJ guidance:
+
+- the revocation or otherwise destruction of decryption keys and/or mechanisms to render data inaccessible or otherwise void through the use of modern cryptography;
+  AND/OR
+- data overwriting methods consisting of at least 3 (three) complete overwrite passes of random data.
```

Comment:
with confirmation via read-back that it's been successful?

Reply:
@cybersquirrel that can be wasteful/pricey, do you think this bit (bold) suffices?

> The base principle
For legislative, regulative, privacy and security purposes, it must be possible to decommission and delete (irreversibly ‘erase’ or ‘destroy’) data *and confirm to a degree of relative confidence it has been completed*